data/language/dutch.txt

```diff
@@ -2382,19 +2382,19 @@ STR_2380    :{SMALLFONT}{BLACK}Kleiner stuk water aanpassen
 STR_2381    :{SMALLFONT}{BLACK}Groter stuk water aanpassen
 STR_2382    :Land
 STR_2383    :Water
-STR_2384    :{WINDOW_COLOUR_2}Je doel:
+STR_2384    :{WINDOW_COLOUR_2}Je doelstelling:
```

Comment:
Dan laat ik 'doelstelling' staan.
In het geval 'segment' denk ik trouwens ook wel dat de meeste basisschoolleerlingen van de bovenbouw dat kennen, en anders is het een mooi woord om daar te leren? Maar daar kom ik later nog wel op terug, als iemand anders ook denkt dat we daar nog even naar moeten kijken?